### PR TITLE
fix: fonts are not being found... this adds the fonts to the build, a…

### PR DESCRIPTION
**Pull request by JasonLantz**
fix: fonts are not being found... this adds the fonts to the build, and commits package-lock.json as per npm recomend...

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="en">
```
Comment (aaronkaka) [inline demo/index.html]: Why?

Comment (JasonLantz) [inline demo/index.html]: didn't change intl settings.... seemed superfluous....

Comment (StommePoes) [inline demo/index.html]: It's not. It's not for Intl settings. It's to tell browsers to expose to assistive tech (or, actually, any robots/scripts reading the page) which  human language this page is written in.

So specifically for the speech synthesis, this tells which language to pronounce the text inside in and which speech rules to follow. Mostly important for English pages being read by screen readers like my JAWS which, due to rules about where I can purchase it, has a default language of Dutch :P

File: webpack.config.js [hunk 2]
```diff
@@ -8,14 +9,18 @@ const main              = `${__dirname}/demo/main.js`;
...
+const fontsDir          = `${__dirname}/node_modules/pearson-elements/dist/fonts/`;
+const fonts             = fs.readdirSync(fontsDir, 'utf-8');
+const fontsList         = fonts.map(font => fontsDir + font);
 
 
 module.exports = {
   entry: {
     demo             : [ demo, demoScss ],
-    dev              : [ elements, icons, main ],
+    dev              : [ elements, icons ],
```
Comment (StommePoes) [inline webpack.config.js]: Didn't Dinuka have main added in so it could be found without direct URLs or something?

File: demo/index.html (hunk 1 above)
Comment (JasonLantz) [inline demo/index.html]: k, i'll put it back....

File: webpack.config.js (hunk 2 above)
Comment (JasonLantz) [inline webpack.config.js]: not sure...

**Pull request merged**